includes/Jobs/GenerateProductFeed.php

```diff
@@ -193,7 +193,18 @@ public function get_plugin_name(): string {
 	 * @return int
 	 */
 	protected function get_batch_size(): int {
-		return 15;
+		/**
+		 * Feed batch size filter.
+		 *
+		 * This filter allows modification of how many items will be processed in one batch during the feed file generation.
+		 * Some, especially big sites with big products catalog, may want to increase this value in order to process the catalog faster.
+		 * This requires careful approach, bumping the value too high may lead to out of memory issues.
```

Comment:
Could make this sentence clearer by stating the tradeoffs. Here's a rough idea, I think you understand the tradeoffs better than me 😁 

- more products per batch:
   - more memory required per batch
   - fewer batches required overall
- fewer products per batch:
  - lower overhead / resources required per batch, increase stability
  - more batches needed, feed generation make take longer overall